src/NServiceBus.Persistence.AzureStorage.ComponentTests/Subscriptions/SuscriptionTestHelper.cs

```diff
@@ -3,29 +3,27 @@
     using System;
     using Microsoft.WindowsAzure.Storage;
     using Microsoft.WindowsAzure.Storage.Table;
-    using Config;
     using Unicast.Subscriptions;
     using Unicast.Subscriptions.MessageDrivenSubscriptions;
 
     public class SubscriptionTestHelper
     {
         internal static ISubscriptionStorage CreateAzureSubscriptionStorage()
         {
-            var config = new AzureSubscriptionStorageConfig();
             var connectionString = AzurePersistenceTests.GetConnectionString();
             var account = CloudStorageAccount.Parse(connectionString);
 
-            var table = account.CreateCloudTableClient().GetTableReference(config.TableName);
+            var table = account.CreateCloudTableClient().GetTableReference(AzureSubscriptionStorageDefaults.TableName);
```

Comment:
Is there a way of overriding the default? How the previous version configured with a custom config, can provide the same value when migrating?